src/OrchardCore.Modules/OrchardCore.ContentFields/Views/TextField-TextArea.Edit.cshtml

```diff
@@ -1,13 +1,15 @@
 @model OrchardCore.ContentFields.ViewModels.EditTextFieldViewModel
 @using OrchardCore.ContentManagement.Metadata.Models
-@using OrchardCore.ContentFields.Settings;
+@using OrchardCore.ContentFields.Settings
+@using OrchardCore.Localization
```

Comment:
Maybe move this using in the `_ViewImports.cshtml` file so you don't have to include in all files ?